### ISSUE TITLE
Release title

### ISSUE THREAD

**Issue by vatterspun** [alexporrello/TWBoilerplateMacros#1]
You might want to label the "Macro Update Utility" release as "Acronym Table Duds List v3" with a link/explanation of how the install_windows.bat function works.  Or update the install text.  I got a bit confused there.

**Comment by alexporrello** [comment]
Yes, the documentation is far from complete. I actually have to re-write most of the read-me, as the install process was greatly simplified with the batch file.
